icons/car.svg

```diff
@@ -9,7 +9,10 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <path d="M14 16H9m10 0h3v-3.15a1 1 0 0 0-.84-.99L16 11l-2.7-3.6a1 1 0 0 0-.8-.4H5.24a2 2 0 0 0-1.8 1.1l-.8 1.63A6 6 0 0 0 2 12.42V16h2" />
-  <circle cx="6.5" cy="16.5" r="2.5" />
-  <circle cx="16.5" cy="16.5" r="2.5" />
+  <path d="M2 11h9c1.1 0 2.7-.2 3.5-.5L16 10" />
```

Comment:
This one feels a bit out of place, how about making it a bit more geometric?
```suggestion
  <path d="M2 11h12.2a2 2 0 0 0 1.386-.586L16 10" />
```

![image](https://github.com/lucide-icons/lucide/assets/17746067/79966ca0-81a9-47d0-999e-81d0cf3ee124)